dev-docs/sanity-workflow.md

```diff
@@ -75,4 +95,4 @@ The following environment variables are required:
 
 If you want to destroy the provisioned infrastructure, set the `cleanup-env` input to `true` when running the workflow. The cleanup step will be executed at the end.
 
-Note: Destroying the environment is irreversible, so use it with caution.
\ No newline at end of file
+Note: Destroying the environment is irreversible, so use it with caution.
```

Comment:
Let's remove this part, I don't want to discourage people from cleaning their cloud envs. They can always redeploy